README.md

```diff
@@ -4,7 +4,7 @@ whatwg-url is a full implementation of the WHATWG [URL Standard](https://url.spe
 
 ## Current status
 
-whatwg-url is currently up to date with the URL spec up to commit [0f5395](https://github.com/whatwg/url/commit/0f53958338bbaec3882f902897873da59ba7e8bd), except that we haven't implemented changes relating to the new UTS #46 revision ([dc9d831](dc9d83106cada9af507bf37dee3973de97b020fd), [b128ba9](https://github.com/whatwg/url/commit/b128ba9111c68ad767b472d77d0ada9ef85366ef)) yet. Work on fixing that gap is in progress in [Sebmaster/tr46.js#11](https://github.com/Sebmaster/tr46.js/pull/11).
```

Comment:
Should update the w3c tests in this commit as well if we're modifying the readme here.